packages/docs/package.json

```diff
@@ -9,34 +9,20 @@
     "dist"
   ],
   "scripts": {
-    "prebuild": "yarn changelog && yarn vendor && yarn micromodal && yarn choices && yarn prism",
-    "build": "hexo g",
-    "start": "yarn build && hexo s",
+    "prebuild": "yarn changelog",
+    "build": "vuepress build",
+    "start": "yarn build && vuepress dev",
     "test": "echo \"Tests have not been implemented for ${npm_package_name}\" && exit 0",
-    "vendor": "mkdir -p themes/nimatron/source/js/vendor && mkdir -p themes/nimatron/source/css/vendor/",
-    "micromodal": "cp ../../node_modules/micromodal/dist/micromodal.min.js themes/nimatron/source/js/vendor/micromodal.min.js",
-    "choices": "cp ../../node_modules/choices.js/public/assets/scripts/choices.min.js themes/nimatron/source/js/vendor/choices.min.js",
-    "prism": "cp ../../node_modules/prismjs/prism.js themes/nimatron/source/js/vendor/prism.js && cp ../../node_modules/prismjs/components/prism-scss.js themes/nimatron/source/js/vendor/prism-scss.js",
-    "changelog": "cp ../odyssey/CHANGELOG.md ./source/getting-started/changelog.md"
+    "changelog": "cp ../odyssey/CHANGELOG.md ./getting-started/changelog.md"
```

Comment:
We should try to follow best practice and let [VuePress' `configureWebpack`](https://vuepress.vuejs.org/config/#configurewebpack) option handle this. That way we can remove the `prebuild` and `changelog` scripts here.

Reply:
I looked in to this, the challenge was that the file cannot merely be copied, it's markdown has to be compiled/transformed in to HTML. Would like to punt on this and address it in the next PR related to docs.